### ISSUE TITLE
Address hylia boilerplate stuff

### ISSUE THREAD

**Issue by bradfrost** [bradfrost/pl-website-eleventy#11]
This project is based off of a boilerplate called [Hylia](https://hylia.website/). We'll want to thoughtfully scrub this from the project, but add a colophon that says this project was built using Hylia as a starter.

**Comment by QuincyGow** [comment]
- In readme, can we delete all mentions? For instance "Get started now by **[deploying Hylia to Netlify.][deploy-to-netlify]**" 
Meaning do we delete all together, or change to "Get started now by **[deploying to Netlify.][deploy-to-netlify]**"? etc...
- For the colophon, would you like this in README?  Maybe create a readme with a log of changes made that include what was deleted/changed?

- In both .json and the MD files, what is system-critical in these files?   It seems to be all Hylia, so I'm just not sure how deep to rip.  Can it all be tossed or are we replacing this with something else?

**Comment by QuincyGow** [comment]
I had this one working perfectly.  I noticed the browser stopped updating with me so I killed the terminal and ran npm start.  Now, I'm getting this error back and the template for Updates won't load or let me change the template.  Trying to get to the bottom of it but would love some help.
<img width="985" alt="Screen Shot 2020-01-22 at 1 29 40 PM" src="https://user-images.githubusercontent.com/48933977/72922592-4c325600-3d1b-11ea-8d72-ee43c732d8f8.png">
